src/app/newcomers/newcomers.component.html

```diff
@@ -103,8 +105,8 @@ <h2>OSS Workflow Diagram</h2>
           aria-hidden="true">
           <div class="modal-dialog modal-notify modal-success" role="document">
             <div class="modal-content">
-              <div class="modal-header colour2">
-                <p class="heading lead colour3">Requirements Analysis</p>
+              <div class="modal-header" style="background-color: #8d2165;">
```

Comment:
Is there a reason why you used an inline style instead of adding it to the stylesheet?

Reply:
@bethanyr This was discussed before. For the modal-header, in the mdBoostrapAngular stylesheet, the colour is defined as green, and if I create a stylesheet - newcomers.component.scss and add `background-color: #8d2165;` there, then it gets overridden by the property defined in mdBoostrapAngular stylesheet. There is a particular order of preference in which Angular is adding the stylesheets. However, the style code written within the style attribute, overrides all other style commands for that DOM element, thus ensuring that the colour of the modal-header is #8d2165 as we wanted and not green.
Is this okay? Or shall I try to look up deeper to find a a solution for this?